portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,167 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>My Portfolio - Saakshi Saxena</h1>
+      <figure>
+        <img src="/my-images/my-pic-1.jpeg" alt="Girl with a pretty background">
```

Comment:
This image link is broken (at least in GitHubs preview), I think you have to remove the leading slash.

Reply:
Done! It should work now.